app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+
+
+def movie_file(command: str):
+    command_list = command.split()
+    if command_list[0] != 'mv':
```

Comment:
use double quotes everywhere

Reply:
To be the same everywhere? Or is there an advantage over single quotes?